### DISCUSSION TITLE
fifc doesn't work after restart of terminal

### DISCUSSION THREAD

**Discussion by Jomik** [gazorby/fifc#37] [Q&A]
Hey, I run `fisher install gazorby/fifc` and then try to complete something. `fzf` / `fifc` pops up as expected to allow me to search and complete.
But after I restart the terminal, I am back to normal fish completions. I have no idea why.. :)
My config is here https://github.com/Jomik/dotfiles/tree/main/.config/fish nothing overly complicated.

Also, how do I show hidden files once searching? (Without just always enabling them)

**Comment by Karen-Abgaryan** [comment]
Hi there, I had the same issue. In my case, the default fifc's keybinding was overwritten with keybindings from `/usr/share/fish/vendor_conf.d/autopair.fish` after restarting the terminal.
I uninstalled the `fish-autopair` package, and now fifc works as expected.
